src/test/java/org/opensearch/security/ssl/SecuritySSLReloadCertsActionTests.java

```diff
@@ -211,6 +365,15 @@ private void updateFiles(String srcFile, String dstFile) {
         FileHelper.copyFileContents(FileHelper.getAbsoluteFilePathFromClassPath(srcFile).toString(), dstFile);
     }
 
+    private JsonNode getNewCertAuthorityUpdatedCertDetailsExpectedResponse(String updateChannel) {
+        String updateKey = (Objects.equals(updateChannel, "http")) ? HTTP_CERTIFICATES_LIST_KEY : TRANSPORT_CERTIFICATES_LIST_KEY;
+        String oldKey = (Objects.equals(updateChannel, "http")) ? TRANSPORT_CERTIFICATES_LIST_KEY : HTTP_CERTIFICATES_LIST_KEY;
+        final var updatedCertDetailsResponse = DefaultObjectMapper.objectMapper.createObjectNode();
+        updatedCertDetailsResponse.set(updateKey, buildCertsInfoNode(NEW_CA_NODE_CERT_DETAILS));
+        updatedCertDetailsResponse.set(oldKey, buildCertsInfoNode(NODE_CERT_DETAILS));
+        return updatedCertDetailsResponse;
```

Comment:
I got kind of confused reading this helper function - can we refactor this for readability? I thought that this was just to help with the dn of either the http or transport certs, but why do we have keys for both http and transport (not just one or the other)?

Reply:
Thanks @derek-ho, agreed! I have refactored this and the existing `getUpdatedCertDetailsExpectedResponse` into a simpler method. Both http and transport keys are required because the expected JSON response returned here is compared with the response from a `GET` to `_opendistro/_security/api/ssl/certs`, which includes both http and transport certs:

```json
{
    "http_certificates_list": [
        {
            "issuer_dn": "CN=Example Com Inc. Signing CA,OU=Example Com Inc. Signing CA,O=Example Com Inc.,DC=example,DC=com",
            "subject_dn": "CN=node-1.example.com,OU=SSL,O=Test,L=Test,C=DE",
            "san": "[[0, [2.5.4.3, node-1.example.com]], [2, localhost], [2, node-1.example.com], [7, 127.0.0.1], [8, 1.2.3.4.5.5]]",
            "not_before": "2023-04-14T13:22:53Z",
            "not_after": "2033-04-11T13:22:53Z"
        }
    ],
    "transport_certificates_list": [
        {
            "issuer_dn": "CN=Example Com Inc. Signing CA,OU=Example Com Inc. Signing CA,O=Example Com Inc.,DC=example,DC=com",
            "subject_dn": "CN=node-1.example.com,OU=SSL,O=Test,L=Test,C=DE",
            "san": "[[0, [2.5.4.3, node-1.example.com]], [2, localhost], [2, node-1.example.com], [7, 127.0.0.1], [8, 1.2.3.4.5.5]]",
            "not_before": "2023-04-14T13:23:00Z",
            "not_after": "2033-04-11T13:23:00Z"
        }
    ]
}
```

This way we can verify that the cert for only one channel was updated.